### PR TITLE
Added All The Flavors 

### PR DESCRIPTION
**Pull request by sunx2**
its a recipe and DIY site

here are 5 presences but the code can generate presence for every url this site has.

### CODE REVIEW

**Comment by sunx2** [comment]
it also address issue https://github.com/PreMiD/Presences/issues/298

File: All The Flavors/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+        "name": "Nekolen",
+        "id": "270125856679002113"
+    },
+    "service": "alltheflavors",
```
Comment (Bas950) [inline All The Flavors/dist/metadata.json]: Name the service the same as the folder name

Comment (sunx2) [reply All The Flavors/dist/metadata.json]: I did it at first but if i put space in the service, it doesn't load in presence launcher

Comment (Bas950) [reply All The Flavors/dist/metadata.json]: image
https://premid.app/store/presences/Google%20Docs

Works on google docs

Comment (sunx2) [reply All The Flavors/dist/metadata.json]: sure then

**Pull request merged**